server/src/models/results/comparePwResult.ts

```diff
@@ -0,0 +1,6 @@
+import { SocoboUser } from "./../database/socobouser";
```

Comment:
Maybe I would replace 'Result' with something like 'Response' or 'ResponseBody' since this kind of objects is mainly used as response body.